### ISSUE TITLE
[Submission form] First accessibility feature cannot be removed using keyboard and screen reader

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wai-evaluation-tools-list#26]
Under "Tool Functionality" > "Features".

- Add at least three features by typing in the corresponding fields and then pressing the "Add" button.
- Put screen reader focus on the "Tool Functionality" heading and down arrow through the added features.

The first added feature doesn't have a "Remove" button". The other subsequently added features do.

Ideally:

- [ ] The first feature should have a "Remove" button just as the others do
- [ ] Every "Remove" button should expose the info of what the feature is to screen readers. When there are many features added that is not apparent. For example, button should say "Remove my first feature", "Remove my second feature", and so on.

**Comment by daniel-montalvo** [comment]
Thanks much @VeraLange 

This is now much better.

> Could you let me know if it's still possible for you to remove the text in the first feature's input field, without a button? If not, we can still add a Remove button, which then would "clear" the field but keep it in the UI.

Yes, I can.

---

There are still some issues when you add more than one feature.

The label for the button "Remove feature x" contains wrong numbering. For the second feature it says "11", for the third it says "21", for the fourth it says "31", etc.

Irrespective of the above, numbers don't help much heere. Can we make it so that it says "Remove [name-of-feature]" instead of numbering? This would be much clearer.

**Comment by VeraLange** [comment]
The way features are added has been changed in the meantime to a simple text field input. Thanks for informing us about these issues anyway, @daniel-montalvo . It'll help us in case we create a similar input for other WAI-pages. 😄
